### DISCUSSION TITLE
apps not workign under wayland  ?

### DISCUSSION THREAD

**Discussion by ghost** [srevinsaju/zap#95] [Q&A] [answered]
whenever i run an app under wayland i got this error 
```
Failed to execute: 'desktop'
Error: 'Failed t execute child 
process "desktop" (No such file 
or directory)'
```
I have multiple of apps installed under zap and none of theme is working, it is probably a problem with environmental variable, any way to fix it ?

**Comment by rywng** [comment, marked as answer]
If you are using terminal to launch desktop files, you should use ```gtk-launch```.
